Upstream MVT runner code

pradeep-raveendranpillai-infosys opened the pull request:
Upstream MVT runner code to rdkcentral repo

pradeep-raveendranpillai-infosys committed:
ONEM-26271: Upstream MVT runner code

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** 96f6ba9ebf0e10ddae002e7b12870e3271116fdf
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/ed66038d9144946429b8315bf02189ab'

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 12 files pending identification.
* Protex Server Path: /home/blackduck/github/MVTRunner/5/rdkcentral/MVTRunner

* Commit: 96f6ba9ebf0e10ddae002e7b12870e3271116fdf

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/765776be47bec194ce7228c49b6b8c6a)'

mhughesacn commented:
Hi Pradeep:

This is a new component and scan resolution would usually take longer, but this one is small:

1.	Standard license files: LICENSE (Apache-2), COPYING (soft link to LICENSE), NOTICE (See e.g. subtec-app lines 1-10), CONTRIBUTING.md (same repo)
2.	crt file(s) – please confirm that this is not a sensitive value, also the .enc file.  If possible, can they have “test” in the name, or an entry in NOTICE saying that the values are non-sensitive.
3.	Missing header on fixtures/stb.py
4.	fixtures/websocket.py contains a key.  Is this sensitive?  If not, please add a comment saying that it is a test value etc.
5.	I presume that http://mvt.onemw.net/ is not a secret URL?
6.	fixtures/stb.py: same comment https://widgets.metrological.com/

If there are any sensitive values here, the repo should be recreated clean.

pradeep-raveendranpillai-infosys committed:
Added more licence info

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/MVTRunner/5/rdkcentral/MVTRunner

* Commit: 228a59692c921879b41a8242fd209dd33fb300b3

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/17ef8ca38706a378904e8db3064f8a59)'

pradeep-raveendranpillai-infosys committed:
Additional changes wrt keys/certs

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/MVTRunner/5/rdkcentral/MVTRunner

* Commit: 891326f2f7f6cb6b755eb4830a2699f34b49ad42

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/d51011cdc1bff34a23e42013f67b8e6a)'

pradeep-raveendranpillai-infosys committed:
Removed unused code

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/MVTRunner/5/rdkcentral/MVTRunner

* Commit: 7d01802a0be5e04dc90ac6f8860e2d08053880d8

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/89f8c6d1a833a60394461c99514d7c03)'

pradeep-raveendranpillai-infosys committed:
Removed mentioning of LGI URL

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/MVTRunner/5/rdkcentral/MVTRunner

* Commit: 6911be5b6db77bc4c7a468fbc4b6f4ca35b84fa3

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/1d936724f4c9a8707bad34c1c72b7fc9)'

pradeep-raveendranpillai-infosys committed:
Updated readme

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/MVTRunner/5/rdkcentral/MVTRunner

* Commit: 5e9694dc77ae8aa7468d9b3d679c8f8801ebe89e

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/dc3930c186152ca3eb1cc15fb64f4adc)'

pradeep-raveendranpillai-infosys commented:
> Hi Pradeep:
> 
> This is a new component and scan resolution would usually take longer, but this one is small:
> 
> 1. Standard license files: LICENSE (Apache-2), COPYING (soft link to LICENSE), NOTICE (See e.g. subtec-app lines 1-10), CONTRIBUTING.md (same repo) - **Added**
> 2. crt file(s) – please confirm that this is not a sensitive value, also the .enc file.  If possible, can they have “test” in the name, or an entry in NOTICE saying that the values are non-sensitive. - **Non sensitive, updated NOTICE**
> 3. Missing header on fixtures/stb.py - **Added**
> 4. fixtures/websocket.py contains a key.  Is this sensitive?  If not, please add a comment saying that it is a test value etc. **Not sensitive.Added comment**
> 5. I presume that http://mvt.onemw.net/ is not a secret URL? - **removed from code**
> 6. fixtures/stb.py: same comment https://widgets.metrological.com/ **removed from code**
> 
> If there are any sensitive values here, the repo should be recreated clean.

mhughesacn commented on LICENSE.md:
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Please rename to LICENSE which is the standard throughout RDK.

pradeep-raveendranpillai-infosys replied:
Done

mhughesacn commented on fixtures/stb.py:
```diff
@@ -0,0 +1,230 @@
+import json
```
This one slipped through - please add header.

pradeep-raveendranpillai-infosys replied:
Added

mhughesacn reviewed (commented):
Two comment please

pradeep-raveendranpillai-infosys committed:
Updated review comments

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/MVTRunner/5/rdkcentral/MVTRunner

* Commit: 105b8ad896fd793efb2d8d96c1f200f99f86f227

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/c678278b36bdfc3da9757ae7caeff595)'

pradeep-raveendranpillai-infosys committed:
Renamed LICENSE.md to LICENSE

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/MVTRunner/5/rdkcentral/MVTRunner

* Commit: 37995a032c0a6dfad2947fa867cfe3be3412b503

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/a0220b9c32655c846a2700fe95dd670b)'

pradeep-raveendranpillai-infosys committed:
Softlink update

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/MVTRunner/5/rdkcentral/MVTRunner

* Commit: 5f102b770db9cda77238823347df36fcd1a1dc8e

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/24f767ff7cf446d662bd845362aeb4ca)'

mhughesacn commented:
Thank you - cleared off in BD
